### PR TITLE
Add TryBooking

### PR DESCRIPTION
**Pull request by zebby-nick**
This will add a new Org and Source for the TryBooking API.

### CODE REVIEW

**Comment by zebby-nick** [comment]
@googlebot I signed it!

File: organizations/trybooking [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pfrisella) [inline organizations/trybooking:1]: File needs to have .json extension.

File: sources/trybooking [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (pfrisella) [inline sources/trybooking:1]: File needs to have .json extension.

File: sources/trybooking [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  "id": "trybooking",
```
Comment (pfrisella) [inline sources/trybooking:2]: ID needs to be uppercase

**Review by pfrisella — CHANGES_REQUESTED** [review]

**Comment by zebby-nick** [comment]
Fixed extensions and changed ID to uppercase.

**Review by pfrisella — APPROVED** [review]
LGTM. Thanks for adding!

**Pull request merged**